Realm/RLMProperty.mm

```diff
@@ -342,6 +342,16 @@ - (instancetype)initSwiftPropertyWithName:(NSString *)name
 
     id propertyValue = [obj valueForKey:_name];
 
+    // temporarily workaround since Objective-C generics used in Swift show up as `@`
```

Comment:
Please make this a `FIXME: `, add information about which versions of Xcode and Swift this problem occurs in, and add a link to the Radar and/or swift.org bug report. It's important to make it easy for a future reader to determine whether this block of code can go away.